README.md

```diff
@@ -14,27 +14,31 @@ This repository aims to provide a base of generic and pre configured monitors fo
 
 ## Getting started
 
-### Terraform
+### Versions
 
-Here is the minimum version required to use these modules of integrations.
+Here is the minimum versions required to use these modules of integrations.
 
-```
+```hcl
 terraform {
-  required_version = ">= 0.12.26"
+  required_providers {
+    datadog = {
+      source = "DataDog/datadog"
+      version = ">= 3.1.0"
+    }
+  }
+  required_version = ">= 0.12.31"
 }
 
 ```
 
-Note: terraform 0.12.26+ is required as it the first version compatible with, but ignoring, the `source` argument for provider requirements introduced in terraform 0.13.0.
+_Note_: if you want to use Datadog provider v2 so you need to use version 3 of this repository.
```

Comment:
```suggestion
_Note_: if you want to use Datadog provider v2, you need to use version 3 of this module.
```

Reply:
it feels weird to speak about "module" whereas we are on the root README which is not a module no?

Reply:
What about:
```suggestion
_Note_: if you want to use Datadog provider v2, you need to use version 3 of the modules in this repository.
```

Reply:
good for me !